web/src/engine/websites/Qiman5.ts

```diff
@@ -0,0 +1,39 @@
+import { Tags } from '../Tags';
+import icon from './Qiman5.webp';
+import { Chapter, DecoratableMangaScraper, Page, type Manga } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchJSON } from './../platform/FetchProvider';
+
+type APIChapters = {
+    data: {
+        name: string,
+        link : string
+    }[]
+}
+
+@Common.MangaCSS(/^{origin}\/[^.]+\.html$/, 'div.info h1.name_mh')
+@Common.MangasMultiPageCSS('/category/tags/6/page/{page}', 'div.item p.title a', 1, 1, 0, Common.AnchorInfoExtractor(true))
+@Common.ImageAjax()
+
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('qiman5', `奇漫屋 (Qiman5)`, 'https://qiman2.com', Tags.Language.Chinese, Tags.Media.Manhua, Tags.Media.Manga, Tags.Media.Manhwa, Tags.Source.Aggregator);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override async FetchChapters(manga: Manga): Promise<Chapter[]> {
+        const mangaId = manga.Identifier.match(/\/(\d+)\.html/)[1];
```

Comment:
This is a code smell as the manga identifier seems to store bloat which is not necessary to identify the manga.

Reply:
I could use a custom MangaInfoExtractor. But MangaCSS use a Label Extractor that doesnt allow us to make a customa Manga ID. So i would need a custom FetchManga.

I'd rather write less code , therefore just making a custom FetchChapters is the most efficient way to go.